### ISSUE TITLE
Add helm-chart deployments to the Makefile

### ISSUE THREAD

**Issue by rasheedamir** [stakater/IngressMonitorController#82]
As suggested by @Anish-Gupta-Bose it will be good to add helm-chart deployments to the Makefile:

```
helm-template:
	helm template kubernetes/chart/ingressmonitorcontroller --values $(HELMVALUES) --name $(HELMNAME)

helm-install:
	helm install kubernetes/chart/ingressmonitorcontroller --values $(HELMVALUES) --name $(HELMNAME)
```

**Comment by Anish-Gupta-Bose** [comment]
Fix Pushed in #88

**Comment by rasheedamir** [comment]
Closed by #88
